circle.yml

```diff
@@ -1,6 +1,10 @@
 machine:
+  environment:
+    REDIS_URL: redis://0.0.0.0:6379/1
   node:
     version: 8.6.0
+  services:
+    - redis
```

Comment:
well isn't that pleasingly simple.

Reply:
I know! If we didn't require a host in the Node source, I doubt I would even have had to provide an environment variable.